src/store/api.js

```diff
@@ -101,8 +101,28 @@ export async function getPaginatedEvents({
   return { items: drops, total };
 }
 
-export async function getEvent(id) {
-  return await fetchPOAPApi(`/events/id/${id}`);
+export async function getDrop(id) {
+  const paginatedResult = await compass.request(PAGINATED_DROPS_QUERY, {
```

Comment:
Does it make sense to specify the type of the paginatedResult?

Reply:
This is js :(